package/yast2-iscsi-client.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jun 18 14:22:05 CEST 2015 - gs@suse.de
+
+- warn only once about non-standard initiator name (bnc #926621)
+- improve help text concerning re-discovery of targets and 
+  start-up mode (bnc #924502)
```

Comment:
I thought space between # and number is forbidden.
Will change it and also change bnc to bsc.